srcs/incs/SyscallWrap.hpp

```diff
@@ -11,6 +12,20 @@ class SyscallWrap {
 						   char *const envp[]);
 		static int closeWr(int fd);
 		static pid_t forkWr(void);
+		static int selectWr(int nfds, fd_set *readfds, fd_set *writefds,
+							  fd_set *exceptfds, struct timeval *timeout);
+		static int socketWr(int domain, int type, int protocol);
+		static int fcntlWr(int fd, int cmd);
+		static int fcntlWr(int fd, int cmd, int64_t arg);
+		static int fcntlWr(int fd, int cmd, struct flock *lock);
```

Comment:
To avoid duplicates, here we could have a variadic function:
```
static int fcntlWr(int fd, int cmd, ...);
```
and use `va_list` to pass the arguments to `fcntl()`

[SO](https://stackoverflow.com/questions/3530771/passing-variable-arguments-to-another-function-that-accepts-a-variable-argument) topic.

Reply:
There is printf(const char*fmt, ...); and then there is vprintf(const char*fmt, va_list ap);
I wouldn't have presumed that you can pass a va_list as a '...'.
But if you say so...

Reply:
> There is printf(const char * fmt, ...); and then there is vprintf(const char * fmt, va_list ap); I wouldn't have presumed that you can pass a va_list as a '...'. But if you say so...

Sorry, my suggestion was wrong, and it will not work.

Reply:
reverted